.golangci.yml

```diff
@@ -70,6 +70,11 @@ issues:
         - gosec
         - perfsprint
         - usestdlibvars
+    # Ignoring gosec G402: TLS MinVersion too low
+    # as the https://pkg.go.dev/crypto/tls#Config handles MinVersion default well.
```

Comment:
Should we move this to be only for the config package, or only for a specific line? Ignoring it globally seems prone to errors.

Reply:
done, is it worth opening an issue to do the same in the core repo?

Reply:
I would think so, yes.